DNN Platform/Website/Providers/DataProviders/SqlDataProvider/09.05.00.SqlDataProvider

```diff
@@ -125,6 +125,158 @@ WHERE p.[Name] IN ('DotNetNuke.Authentication',
+    @SchedulerInstance UNIQUEIDENTIFIER,
+    @BatchSize         INT
+AS 
+BEGIN
+    -- reset possibly remaining records from any previous run of this SchedulerInstance:
+    UPDATE {databaseOwner}[{objectQualifier}CoreMessaging_MessageRecipients]
+       SET [EmailSchedulerInstance] = Null,
+           [LastModifiedOnDate]     = GetDate()
+     WHERE [EmailSchedulerInstance] = @SchedulerInstance
+       AND [EmailSent] = 0 AND [Read] = 0 AND [Archived] = 0;
+       
+    -- reset possibly remaining outdated records from other instances:
+    UPDATE {databaseOwner}[{objectQualifier}CoreMessaging_MessageRecipients]
+     SET   [EmailSchedulerInstance] = Null
+     WHERE [EmailSent] = 0 AND [Read] = 0 AND [Archived] = 0
+       AND [EmailSchedulerInstance] Is Not Null AND [LastModifiedOnDate] < DateAdd(hh, -2, GetDate());
+       
+
+    -- mark messages for dispatch, so they won't be handled by another SchedulerInstance:
+    UPDATE TOP (@BatchSize) R
+     SET   [EmailSchedulerInstance] = @SchedulerInstance,
+           [LastModifiedOnDate]     = GetDate()
+     FROM       {databaseOwner}[{objectQualifier}CoreMessaging_MessageRecipients] AS R 
+     INNER JOIN {databaseOwner}[{objectQualifier}CoreMessaging_Messages]          AS M ON R.MessageID = M.MessageID
+     LEFT  JOIN {databaseOwner}[{objectQualifier}CoreMessaging_UserPreferences]   AS P ON R.UserID    = P.UserID    AND M.PortalID = P.PortalID
+     WHERE R.[EmailSent] = 0 AND R.[Read] = 0 AND R.[Archived] = 0 AND EmailSchedulerInstance IS NULL
+       AND CASE 
+            WHEN M.NotificationTypeID IS Null 
+            THEN IsNull(P.[MessagesEmailFrequency],      0) -- direct mails are sent immediately by default
+            ELSE IsNull(p.[NotificationsEmailFrequency], 2) -- notifications are sent as daily digest by default
+           END = 0;
+
+    SELECT M.[PortalID],
+           M.[NotificationTypeID],
+           M.[To],
+           M.[From],
+           M.[Subject],
+           M.[Body],
+           M.[SenderUserID],
+           M.[ExpirationDate],
+           M.[Context],
+           R.[RecipientID],
+           R.[MessageID],
+           R.[UserID]
+     FROM  {databaseOwner}[{objectQualifier}CoreMessaging_MessageRecipients] R
+     JOIN  {databaseOwner}[{objectQualifier}CoreMessaging_Messages]          M ON R.MessageID = M.MessageID
+     WHERE [EmailSent] = 0 -- Filter these columms 4 to use proper index
+       AND [Read]      = 0
+       AND [Archived]  = 0
+       AND [EmailSchedulerInstance] = @SchedulerInstance
+     ORDER BY --[PortalID],
+              [UserID],
+              [RecipientID]
+END; -- Procedure
+GO
+
+
+IF EXISTS (SELECT * FROM sys.Procedures WHERE object_id = OBJECT_ID(N'{databaseOwner}[{objectQualifier}CoreMessaging_GetNextMessagesForDigestDispatch]'))
+    DROP PROCEDURE {databaseOwner}[{objectQualifier}CoreMessaging_GetNextMessagesForDigestDispatch];
+GO
+
+CREATE PROCEDURE [dbo].[CoreMessaging_GetNextMessagesForDigestDispatch]
```

Comment:
`databaseOwner` and `objectQualifier` were needed here. Because of this, the Scheduler in 9.5.0 fails to execute this stored procedure.